### DISCUSSION TITLE
Creating a new collection type without downtime?

### DISCUSSION THREAD

**Discussion by brijmcq** [strapi/strapi#7514] [Legacy Topics]
When adding a new collection type, server needs to restart. How do you handle this in production where downtime is something you can't afford. One alternative I could think of is to do this on off-peak hours. I'm wondering how you guys do it.

**Comment by biapar** [comment]
I think that the strapi on production server is the production version built with npx strapi build command.
The changes must be done on dev machine. So after launch build command and move the dist folder to production server.

**Reply by brijmcq** [reply]
thanks for your reply @biapar . also I just saw this doc https://strapi.io/documentation/3.0.0-beta.x/getting-started/troubleshooting.html#why-can-t-i-create-or-update-content-types-in-production-staging. will close this now

**Reply by biapar** [reply]
Thx
